src/diffusers/__init__.py

```diff
@@ -34,6 +40,7 @@
         UNet2DModel,
         VQModel,
     )
+    from .models.modeling_utils import ModelMixin
```

Comment:
```suggestion
    from .models import ModelMixin
```

I think `.modeling_utils` is not necessary here.

Reply:
Nice catch!